clang/lib/CodeGen/CGExpr.cpp

```diff
@@ -4693,6 +4694,17 @@ LValue CodeGenFunction::EmitLValueForLambdaField(const FieldDecl *Field,
     else
       LambdaLV = MakeAddrLValue(AddrOfExplicitObject,
                                 D->getType().getNonReferenceType());
+
+    // Make sure we have an lvalue to the lambda itself and not a derived class.
+    auto *ThisTy = D->getType().getNonReferenceType()->getAsCXXRecordDecl();
+    auto *LambdaTy = cast<CXXRecordDecl>(Field->getParent());
+    if (ThisTy != LambdaTy) {
+      const CXXCastPath &BasePathArray = getContext().LambdaCastPaths.at(MD);
+      Address Base = GetAddressOfBaseClass(
+          LambdaLV.getAddress(*this), ThisTy, BasePathArray.begin(),
+          BasePathArray.end(), /*NullCheckValue=*/false, SourceLocation());
```

Comment:
error: too many arguments to function call, expected 0, have 1
          LambdaLV.getAddress(*this), ThisTy, BasePathArray.begin(),
          ~~~~~~~~~~~~~~~~~~~ ^~~~~

https://lab.llvm.org/buildbot/#/builders/225/builds/36615/steps/6/logs/stdio

Reply:
Already on it

Reply:
Fixed

Reply:
https://github.com/llvm/llvm-project/commit/da88e7fbd74cef33411bb5115f20e4b474d2d8b1